web-console/5-components/order-from-catalog.md

```diff
@@ -54,51 +46,58 @@ There are 3 possible steps in this process: Plan, Configuration and Results.
 
 Post 3.6, an additional Bind Step will be implemented.
 
-### Step 1 - Plan
+### Information Step
 
-### User selects a plan
+![template](img/information-step.png)
 
-#### Catalog item has a single version
-![template](img/plan-step.png)
+1. Catalog Item Icon
+	- metadata.imageURL from ![OpenServiceBroker Service Metadata](https://github.com/openservicebrokerapi/servicebroker/blob/master/profile.md#service-metadata-fields)
```

Comment:
It's trying to render these links as images. Removing the "!" from the front should do the trick

Reply:
web-console/5-components/img/plan-step.png